docs/sources/network/quickstart.md

```diff
@@ -0,0 +1,213 @@
+src.name=local-path-provisioner-7577fdbbfb-g6b7d src.namespace=local-path-storage
+dst.name=kubernetes dst.namespace=default k8s.src.host.ip=172.18.0.2
+k8s.src.host.name=kind-control-plane k8s.dst.namespace=default k8s.dst.name=kubernetes
+k8s.dst.owner.type=Service k8s.src.namespace=local-path-storage
+k8s.src.name=local-path-provisioner-7577fdbbfb-g6b7d k8s.src.type=Pod
+k8s.src.owner.name=local-path-provisioner k8s.src.owner.type=Deployment
+k8s.dst.type=Service k8s.dst.owner.name=kubernetes
+```
+
+For further information on the above attributes, consult the [network metrics documentation]({{< relref "./_index.md" >}}).
+
+## Export OpenTelemetry metrics
+
+After you have confirmed that network metrics are being collected, configure Beyla to export the metrics in OpenTelemetry
+format to an OpenTelemetry endpoint.
+
+{{% admonition type="note" %}}
+Prometheus exporting for network metrics is not currently supported.
+{{% /admonition %}}
+
+Beyla works with any OpenTelemetry endpoint, for this quickstart we use the OpenTelemetry endpoint in Grafana Cloud. You can get a [Free Grafana Cloud Account at Grafana's website](/pricing/).
+
+To get your stack's OpenTelemetry endpoint, login to the Grafana Cloud Portal, and click **Configure** under the **OpenTelemetry** section.
+
+![OpenTelemetry Grafana Cloud portal](https://grafana.com/media/docs/grafana-cloud/beyla/quickstart/otel-cloud-portal-box.png)
+
+Under **Password / API token**, click **Generate now** and follow the instructions to create an API token.
+
+The **Environment Variables** will be populated with a set of standard OpenTelemetry environment variables which will provide the connection endpoint and credentials information for Beyla.
+
+![OTLP connection headers](https://grafana.com/media/docs/grafana-cloud/beyla/quickstart/otlp-connection-headers.png)
+
+Copy the value of `OTEL_EXPORTER_OTLP_HEADERS` environment variable and paste it as a Kubernetes secret (and deploy it):
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-secret
+type: Opaque
+stringData:
+  otlp-headers: "Authorization=Basic MzQ3NTp....."
+```
+
+Now add the `OTEL_EXPORTER_OTLP_HEADERS` and reference this secret as the variable value.
+
+Also Add `OTEL_EXPORTER_OTLP_ENDPOINT` and its value as an environment variable to the Beyla container in the Kubernetes manifest. The `env` section of the `beyla` container in the manifest from the start of this document should look like:
+
+```yaml
+          env:
+            - name: BEYLA_CONFIG_PATH
+              value: "/config/beyla-config.yml"
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: "https://otlp-gateway-prod-eu-west-0.grafana.net/otlp"
+            - name: OTEL_EXPORTER_OTLP_HEADERS
+              valueFrom:
+                secretKeyRef:
+                  key: otlp-headers
+                  name: grafana-secret
+```
+
+## Select metrics attributes to reduce cardinality
```

Comment:
Are we going to add content for these two sections?

Reply:
Yes, this is a WIP PR.